import "MsgHead.proto";    
message MsgC2GateLoginReq
{	
	required string id = 1;
	required string password = 2;
}


message MsgC2GateCreateAcountReq
{	
	required string id = 1;
	required string name = 2;
	required string password = 3;
}

message MsgC2GateLoginMacReq
{
	required string Mac = 1;
}

message MsgBindMailReq
{
	required string Mac = 1;
	required string name = 2;
	required string password = 3;
}
